packages/server/lib/project-base.ts

```diff
@@ -446,25 +425,17 @@ export class ProjectBase<TServer extends Server> extends EE {
     specs,
     config,
   }: {
-    specs: Cypress.Cypress['spec'][]
+    specs: FoundSpec[]
     config: Cfg
   }) {
-    const specsStore = new SpecsStore(config, this.testingType)
+    const specsStore = new SpecsStore()
 
     const startSpecWatcher = () => {
-      return specsStore.watch({
```

Comment:
How are specs watched and the specs list updated without starting a watcher?

Reply:
This is moving out of `project-base.ts` into the data context. I don't know if this is implemented yet.

This change does break `runner-ct`'s inline spec list, but that's okay, since we are well and truly using the unified runner ass the main runner now. I removed most of the spec store code, since we no longer need this - we are serving the specs via GraphQL now, not the spec store.